volume/driver/overlay_linux.go

```diff
@@ -78,21 +92,99 @@ func (driver *OverlayDriver) CreateCopyOnWriteLayer(path string, parent string)
 		workDir,
 	)
 
+	fmt.Println(path, opts)
```

Comment:
I guess this is debugging a leftover 🤔

Reply:
nah, this is a crucial step for booting up baggageclaim.